src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Mod.cs

```diff
@@ -61,7 +62,7 @@ public override bool CheckInvocation(TexlNode[] args, DType[] argTypes, IErrorCo
             Contracts.AssertValue(errors);
             Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
 
-            var fValid = base.CheckInvocation(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
```

Comment:
>base. [](http://example.com/codeflow?start=25&length=5)

nit - what's the significant of this change?

Reply:
it was needed to show the intent, avoid a loop but now that we use a different API, it's no more needed

Reply:
I see .. the signature change (adding TexlBinding param) impacts this.